design/js/main.js

```diff
@@ -343,6 +343,31 @@ function initRigidSearch() {
   }
 }
 
+window.Cookies = {
```

Comment:
I see we're already doing some stuff with cookies manually, so this may not be desirable.